core/jazz_metrics/components/metrics.json

```diff
@@ -123,132 +166,298 @@
+            "Statistics": "Min",
+            "Unit": "Milliseconds"
+          },
+          {
+            "Label": "Target Response Time",
+            "MetricName": "target_response_time",
+            "Statistics": "Avg",
+            "Unit": "Milliseconds"
+          },
+          {
+            "Label": "Target Response Time",
+            "MetricName": "target_response_time",
+            "Statistics": "Max",
+            "Unit": "Milliseconds"
+          },
+          {
+            "Label": "Target Response Time",
+            "MetricName": "target_response_time",
+            "Statistics": "Min",
+            "Unit": "Milliseconds"
+          },
+          {
+            "Label": "Request Size",
+            "MetricName": "request_size",
+            "Statistics": "Avg",
+            "Unit": "Bytes"
+          },
+          {
+            "Label": "Request Size",
+            "MetricName": "request_size",
+            "Statistics": "Max",
+            "Unit": "Bytes"
+          },
+          {
+            "Label": "Request Size",
+            "MetricName": "request_size",
+            "Statistics": "Min",
+            "Unit": "Bytes"
+          },
+          {
+            "Label": "Response Size",
+            "MetricName": "response_size",
+            "Statistics": "Avg",
+            "Unit": "Bytes"
+          },
+          {
+            "Label": "Response Size",
+            "MetricName": "response_size",
+            "Statistics": "Max",
+            "Unit": "Bytes"
+          },
+          {
+            "Label": "Response Size",
+            "MetricName": "response_size",
+            "Statistics": "Min",
+            "Unit": "Bytes"
+          },
+          {
+            "Label": "Response Processing Latency",
+            "MetricName": "response_processing_latency",
+            "Statistics": "Avg",
+            "Unit": "Milliseconds"
+          },
+          {
+            "Label": "Response Processing Latency",
+            "MetricName": "response_processing_latency",
+            "Statistics": "Max",
+            "Unit": "Milliseconds"
+          },
+          {
+            "Label": "Response Processing Latency",
+            "MetricName": "response_processing_latency",
+            "Statistics": "Min",
+            "Unit": "Milliseconds"
+          },
+          {
+            "Label": "Request Processing Latency",
+            "MetricName": "request_processing_latency",
+            "Statistics": "Avg",
+            "Unit": "Milliseconds"
+          },
+          {
+            "Label": "Request Processing Latency",
+            "MetricName": "request_processing_latency",
+            "Statistics": "Max",
+            "Unit": "Milliseconds"
+          },
+          {
+            "Label": "Request Processing Latency",
+            "MetricName": "request_processing_latency",
+            "Statistics": "Min",
+            "Unit": "Milliseconds"
+          }
+        ],
+        "dimensions": ["apiproxy"]
```

Comment:
Can we fix these whitespace changes?

Reply:
After applying beautify Whitespace got added.

Reply:
After applying beautify Whitespace got added.

Reply:
But that just messes the code annotation/blame history. If it is not too much effort, can u remove those manually? Maybe Sini had similar issues.

Reply:
Done